### DISCUSSION TITLE
Avoid pop-up asking to share the srceenshot with normcap

### DISCUSSION THREAD

**Discussion by louis-vgn** [dynobo/normcap#331] [Ideas]
I’m not sure if it is an issue only on my computer or not, but the application could take a whole new level if there was no pop-up asking every time if I *really* want to share the screenshot with norm cap or not (see picture).

I’m running Ubuntu Jammy. 

One may say that it’s better to have it because then you can tweak the type of screenshot with the option, but those options may not be the most necessary, and the pop-up is more annoying. 

![image](https://user-images.githubusercontent.com/75619262/209417499-2f0036b1-92cd-43bc-89a9-d3c9e891c45b.png)

Otherwise, great work, love it. Very useful.

**Comment by dynobo** [comment]
@louis-vgn , no, it is not just you. It's a security limitation by Wayland and Free Desktop Portal. The details are written up in #264 (which you might want to subscribe for updates) , but the GIST is: proper screenshot permissions already got implemented and are shipped with newer Linux distributions. E.g. on my Arch Linux I do not have this limitation anymore. I don't know when the change arrives in Ubuntu (I haven't yet checked 22.10).

Does this help you?

**Reply by louis-vgn** [reply]
Absolutely! Sorry I didn't see the issue before bothering with the remark.

Thanks a lot for the answer though.
